### PR TITLE
[MOB-2555] Update trees planted by Ecosia string

### PR DESCRIPTION
**Pull request by d4r1091**
[MOB-2555]

## Context

A small update is required after a clarification from PMs with regards to some strings to update and align to Android (instead of previously confirmed the opposite).

## Approach

Update translations.

### Checklist

- [x] I updated only the [english localization source files](Client/Ecosia/L10N/es.lproj) if needed

[MOB-2555]: https://ecosia.atlassian.net/browse/MOB-2555?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: Client/Ecosia/L10N/en.lproj/Ecosia.strings [hunk 1]
```diff
@@ -153,8 +153,8 @@
 "Trees planted" = "Trees planted";
 "Before ..." = "Before ...";
 "After" = "After";
-"trees planted by the Ecosia community" = "trees planted by the Ecosia community";
-"Trees planted by the Ecosia community" = "Trees planted by the Ecosia community";
+"trees planted by Ecosia" = "trees planted by Ecosia";
+"Trees planted by Ecosia" = "Trees planted by Ecosia";
```
Comment (lucaschifino) [inline Client/Ecosia/L10N/en.lproj/Ecosia.strings:157]: Don't you need to update the place that uses them given the key updated?

Comment (d4r1091) [reply Client/Ecosia/L10N/en.lproj/Ecosia.strings:157]: Of course I do! 🤣 
Too many back and forth with Firefox Upgrade already! Will update 🙏

File: Client/Ecosia/L10N/String.swift [hunk 2]
```diff
@@ -159,8 +159,8 @@ extension String {
         case sustainableShoes = "sustainable shoes"
         case before = "Before ..."
         case after = "After"
-        case treesPlantedByTheCommunity = "trees planted by the Ecosia community"
-        case treesPlantedByTheCommunityCapitalized = "Trees planted by the Ecosia community"
+        case treesPlantedByEcosia = "trees planted the Ecosia"
```
Comment (lucaschifino) [inline Client/Ecosia/L10N/String.swift:162]: 🙈 

```suggestion
        case treesPlantedByEcosia = "trees planted by Ecosia"
```

**Comment by d4r1091** [comment]
@lucaschifino thanks for the careful look 🤦

**Review by lucaschifino — APPROVED** [review]

**Pull request merged**